tests/e2e/pageObjects/browser-page.ts

```diff
@@ -556,6 +558,16 @@ export class BrowserPage {
         await t.click(this.confirmDeleteKeyButton);
     }
 
+    /**
+     * Delete keys by their Names
+     * @param keyNames The names of the key array
+     */
+     async deleteKeysByNames(keyNames: string[]): Promise<void> {
+        for(const name of keyNames) {
+            await this.deleteKeyByName(name);
```

Comment:
I didn't add these console.log prints, because they are not visible in CI reporter logs
and besides, it can always be added manually when debugging locally